index.test.js

```diff
@@ -216,6 +216,20 @@ describe('validate-commit-msg.js', function() {
       expect(errors).to.deep.equal(['INVALID COMMIT MSG: subject does not match subject pattern!']);
       expect(logs).to.deep.equal([msg]);
     });
+
+    it('should lowercase type when autoFix is true and make it valid', function() {
+      m.config.autoFix = true;
```

Comment:
Good question! In general I think tests could use some refactoring. I am not in favour of the side effects of `it`s in the global config. I had to reset `subjectPattern` to make the test pass. We can start with the `autoFix` attribute.